runtime/interop/generate_keys.go

```diff
@@ -5,6 +5,8 @@ import (
 	"math/big"
 	"sync"
 
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+
```

Comment:
```suggestion
	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
```

And goimports / gofmt please